extensions/wikia/UserLogin/css/UserLogin.scss

```diff
@@ -75,3 +75,7 @@ $input-width: 230px;
 		display: none;
 	}
 }
+
+.hidden {
```

Comment:
Hey, are you considering to put this class into a shared file, according to your question in https://wikia-inc.atlassian.net/wiki/questions/19791892/global-css-rules ? That would be great and we could all benefit from that I guess :)